File: WebNovelApp/app/src/main/java/com/sinhvien/webnovelapp/models/Tag.kt
package com.sinhvien.webnovelapp.models

data class Tag(
    val Id: Int = 0,
    val Name: String = "",
    val Description: String? = null,
    val Color: String? = null,
    val IsActive: Boolean = true,
    val CreatedAt: String? = null
)

File: DarkNovel/DarkNovel/Models/Tag.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text.Json.Serialization;

namespace DarkNovel.Models
{
    public class Tag
    {
        public int Id { get; set; }

        [Required]
        [MaxLength(50)]
        public string? Name { get; set; }  // Add ?

        [MaxLength(255)]
        public string? Description { get; set; }  // Add ?

        [MaxLength(7)]
        public string? Color { get; set; }  // Add ?

        public bool IsActive { get; set; } = true;
        public int? CreatedBy { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.Now;

        // Navigation Properties
        public virtual ICollection<NovelTag>? NovelTags { get; set; }  // Add ?
    }
}